### PR TITLE
stella: 4.1.1 -> 6.1.1

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by Emantor — APPROVED** [review]
- [x] build package
```
$ nix build --no-link --keep-going --option build-use-sandbox relaxed -f /home/phoenix/.cache/nixpkgs-review/pr-84611/build.nix
[3 built, 1 copied (1.6 MiB), 0.2 MiB DL]
https://github.com/NixOS/nixpkgs/pull/84611
1 package built:
stella
```
- [x] output binary works and lets me play space invaders 👾👾👾

I think this derivation would vastly profit from using `enableParallelBuilding = true;`, since the compilation takes quite long on a single core. If the stella build system is able to cope with it, you should consider this option.

**Comment by AndersonTorres** [comment]
@Emantor done

File: pkgs/misc/emulators/stella/default.nix [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+    owner = "stella-emu";
+    repo = "stella";
+    rev = version;
+    sha256 = "1iwhslrkq887v035j68lhblybww8r792515rp2m5qzmdgnjzsvbb";
   };
 
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ SDL2 ];
+  
+  enableParallelBuild = true;
```
Comment (OPNA2608) [inline pkgs/misc/emulators/stella/default.nix:19]:
```suggestion
  enableParallelBuilding = true;
```
https://nixos.org/nixpkgs/manual/#ssec-stdenv-attributes

**Review by Emantor — APPROVED** [review]
With `enableParallelBuilding`it now works correctly and builds much faster.

**Pull request merged**